Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
> The method using this property is static too.

I know…  there is the problem, maybe this should be changed too. IMO static is very often bad design because it prevents proper injection.